static/styles/settings.scss

```diff
@@ -1661,7 +1661,7 @@ body:not(.night-mode) #settings_page .custom_user_field .datepicker {
 
 @supports (-moz-appearance: none) {
```

Comment:
Is the fact that this is `-moz-appearance` an issue?

Reply:
No, this block is specifically checking for Firefox and stylelint is fine with that.